src/entropic/sinkhorn.jl

```diff
@@ -290,16 +302,18 @@ function sinkhorn_divergence(
     alg::Sinkhorn=SinkhornGibbs(),
     algμ::Sinkhorn=SymmetricSinkhornGibbs(),
     algν::Sinkhorn=SymmetricSinkhornGibbs();
-    regularization=nothing,
+    regularization=true,
```

Comment:
The default should be `false`, no?

Reply:
Depending on which approach one follows :shrug: Feydy et al. included the regularization terms, Genevay et al. not (at least that's the way it's described in their papers).

Reply:
I don't really know/have a preference. Although with the current "OT" notation we've used I'd say `regularization = true` is most consistent.

Reply:
I think we could go with true, but I think we should emphasize it, since the default for sinkhorn is false.

Reply:
Oops sorry my mistake, I forgot that sinkhorn uses false by default. Probably best to go with regularization=false. @devmotion ?

Reply:
Hmm I'm not sure. I agree that it would be more consistent to use `regularization=false` (i.e., to not specify it at all it seems). But then we would deviate from POT and would not implement the definition in Feydy et al. but Genevay et al.

Reply:
Latest commit sets `regularization = false` now. Probably not a huge problem to deviate from POT.